walkthroughs/2-fuse-aggregator-and-api-management/walkthrough.adoc

```diff
@@ -299,7 +299,7 @@ Verify that you followed each step in the procedure above.  If you are still hav
 
 . Select the *Update & test in Staging Environment* button at the bottom again.
 
-. Check that a success message is displayed, and a green line along the left side of the page.
+. Is a success message is displayed and a green line along the left side of the page?
```

Comment:
Minor typo here `Is a success message is displayed`

Reply:
fixed